src/styles/blocks/header.scss

```diff
@@ -0,0 +1,19 @@
+.header {
+  padding: 0 50px;
+  min-width: 850px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  &__link {
+    width: 40px;
+    height: 40px;
+  }
+
+  &__logo {
+    width: 100%;
+    height: 100%;
+  }
```

Comment:
Useless

Reply:
Agree with you in this particular case becasue logo has same size 40x40px.